### ISSUE TITLE
AppImage loads wrong Qt resulting in symbol lookup error

### ISSUE THREAD

**Issue by probonopd** [OpenShot/openshot-qt#159]
![cf7-f2fuaaep0rs png large](https://cloud.githubusercontent.com/assets/2480569/14583812/a821e5f0-042d-11e6-95d1-83def2df8a30.png)

Source: https://twitter.com/JoeRessington/status/721451685420408835

**Comment by probonopd** [comment]
This can be done at AppImage creation time to prevent this:

```
# Workaround for:
# undefined symbol: g_type_check_instance_is_fundamentally_a
# Function g_type_check_instance_is_fundamentally_a was introduced in glib2-2.41.1
# Bundle libglib-2.0.so.0 - TODO: find a better solution, e.g., downgrade libglib-2.0 at compile time
cp $(ldconfig -p | grep libglib-2.0.so.0 | cut -d ">" -f 2 | xargs) ./usr/lib/
# The following come with glib2 and probably need to be treated together:
cp $(ldconfig -p | grep libgio-2.0.so.0 | cut -d ">" -f 2 | xargs) ./usr/lib/
cp $(ldconfig -p | grep libgmodule-2.0.so.0 | cut -d ">" -f 2 | xargs) ./usr/lib/
cp $(ldconfig -p | grep libgobject-2.0.so.0 | cut -d ">" -f 2 | xargs) ./usr/lib/
cp $(ldconfig -p | grep libgthread-2.0.so.0 | cut -d ">" -f 2 | xargs) ./usr/lib/
```

**Comment by EJCFox** [comment]
I currently have this problem. Does this mean I have to create the appimage myself? Or just run these commands before launching it?...

**Comment by probonopd** [comment]
This needs to be done at AppImage creation time.

**Comment by probonopd** [comment]
See https://github.com/OpenShot/openshot-qt/issues/298

**Comment by DylanC** [comment]
@probonopd  - This issue seems to be resolved in #298. Can this issue be closed?

**Comment by DylanC** [comment]
Resolved in #298
